build.yaml

```diff
@@ -14,7 +14,7 @@ cassandra:
   - 2.1
   - 2.2
   - 3.0
-  - 3.10
+  - 3.7
```

Comment:
That could really be what was happening, now we're trying to run against 3.11, nevertheless I've done the same as in your link and will pass as strings all the versions.